packages/merge-on-green/src/app.ts

```diff
@@ -16,4 +16,6 @@ import {GCFBootstrapper} from 'gcf-utils';
 import appFn from './merge-on-green';
 
 const bootstrap = new GCFBootstrapper();
-module.exports['merge_on_green'] = bootstrap.gcf(appFn);
+module.exports['merge_on_green'] = bootstrap.gcf(appFn, {
+  background: false,
```

Comment:
I don't have a great understanding of what `background: false` really does to be honest.  Could I trouble you for a somewhat wordy explanation here, and the other places we use it?

Reply:
added comments 👍